pages/about.tsx

```diff
@@ -42,21 +44,26 @@ export default function AboutPage() {
           <Button href="#our-team">Our Team</Button>
         </ButtonGroup>
       </SectionContainer>
+      <SectionContainer sx={{ margin: 'auto', mb: 4, maxWidth: '65rem' }} maxWidth={false}>
+        <Typography variant="h3">{aboutContent.ourMission.header}</Typography>
+      </SectionContainer>
 
       <SectionContainer
         id="our-story"
-        sx={{ margin: 'auto', maxWidth: '772px' }}
+        sx={{ margin: 'auto', maxWidth: '65rem' }}
         maxWidth={false}
       >
-        <Typography variant="h2">{aboutContent.ourStory.header}</Typography>
-        <Typography variant="h3">{aboutContent.ourStory.body}</Typography>
+        <Typography variant="h2" sx={{ color: '#2f3554', mt: 4, textAlign: { xs: 'center', sm: 'left' } }}>{aboutContent.ourStory.header}</Typography>
```

Comment:
This header font color is different than the others, can you please make them all match? I tried to do this quickly but am not home and it is not working out for me on a non-dev computer lol.

Reply:
Hi Eleni, You are right. I changed it because I saw that the FIGMA file had that color but I will change it so everything matches.